README.md

```diff
@@ -101,17 +101,29 @@ Node via an environment variable, as well as a valid kubeconfig.  For example:
 
 ## Running the relay with policies enabled
 
-You can enable active policying of containers by passing the
-`--policy <policy-name>` commandline option to the relay. For instance,
+You can enable active policying of containers by using an appropriate ConfigMap
+or a configuration file and setting the `Active` field of the `policy` section
```

Comment:
AFAIU you cannot dynamically change the active policy(?) If so, this could be mentioned

Reply:
Added  a note about this to README.md.